Description: remove fullscreen
 the app start only in fullscreen
 that could be annoying, so we start in windowed mode
Author: Nicolas HOUDELOT <nicolas@demosdebs.org>
Applied-Upstream: no
Last-Update: 2020-04-24
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/buggy_metaballs.cpp
+++ b/buggy_metaballs.cpp
@@ -82,7 +82,7 @@
 glTexParameteri(d, GL_TEXTURE_MIN_FILTER, GL_LINEAR o glEnable(d o 
 } u am(u *e, t *h, g j) { static g a=0;a++; m d[]={.123,.138,.155,.165,.185,.196,.220,.233,2,.03};
 while(j--) h[j]=(sin(j*d[C(0,a%60,0)%10])+1)*128 ; } g main()
-{ SDL_Event a; SDL_Init(32 o SDL_SetVideoMode(640, 480, 32, 2|SDL_FULLSCREEN o 
+{ SDL_Event a; SDL_Init(32 o SDL_SetVideoMode(640, 480, 32, 2 o
  m c[][4]={{ .1, .1, -1, 0},{ .8, -.2, -.5, 0}, { 1, 1, 1}, { .3, .3, .2},	{ .3, .1, .3}};
 glDisable( GL_DEPTH_TEST o g d[]={GL_BLEND,GL_COLOR_MATERIAL,GL_LIGHTING,GL_TEXTURE_2D};
 f(a,3)glEnable(d[a] o } P( o glShadeModel(GL_SMOOTH o 
